scss/_header.scss

```diff
@@ -0,0 +1,138 @@
+@import "variable";
+
+.header {
+  width: 100%;
+  margin: auto;
+  background-color: #f8f9fa;
+  padding: 10px 0;
+}
+
+.navbar {
+  width: 90%;
+  margin: auto;
+  display: flex;
+  justify-content: space-between;
+
+  .logo {
+    width: 30%;
+    display: flex;
+    align-items: flex-start;
+
+    .navbar-brand {
+      width: 100%;
+      margin: auto;
+    }
+  }
+}
+
+.navbar-nav {
+  align-items: center;
+  margin-top: 1.5%;
+
+  .nav-item {
+    padding: 0px 15px;
+    border-radius: 20px;
+    font-size: 95%;
+
+    &:hover {
+      background-color: $bg-color;
+    }
+  }
+}
+
+.navbar-light 
+.navbar-nav 
+.active {
+
+  .nav-link {
+    font-family: "Open Sans", sans-serif;
```

Comment:
những đoạn dùng font family như này thì khai báo ở thẻ body thôi là toàn project sẽ được set font như vậy